packages/iframe-execution-environment-service/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mm-snap/iframe-execution-environment-service",
-  "version": "0.0.6",
+  "version": "0.0.8",
```

Comment:
lol what happened here with the version

Reply:
ah ok, it only updates package versions that have changed it looks like

Reply:
This is working as expected! There was no version `7` of this package, so it goes straight to the latest / current version of the monorepo (`8`). The alternative to "omitting" versions in that manner is to always bump every package in the monorepo, and I don't think that's preferable. 🤔